docs/application-connector/docs/08-04-register-secured-api.md

```diff
@@ -51,3 +53,46 @@ This is an example of the `api` section of the request body for an API secured w
             },
         }  
 ```
+
+## Register an API secured with generated client certificates
+
+To register an API and secure it with generated client certificates, you must add the `credentials.certificateGen` object to the `api` section of the service registration request body.
+You must include this field:
+
+| Field   |  Description |
+|:----------:|:------|
+| **commonName** |  Name of the generated certificate.  |
+
+This is an example of the `api` section of the request body for an API secured with generated client certificates:
```

Comment:
Please explain that it is marking an API as protected by the client cert and additionally for your convenience we are generating client certificate and key which might be replace in future.